frontend/src/store/interview/actions.ts

```diff
@@ -98,11 +98,22 @@ const getOtherByInterviewId = createAsyncThunk<
   },
 );
 
+const handleUserHasNoPermission = createAsyncThunk<
+  void,
+  void,
+  AsyncThunkConfig
+>(ActionType.HANDLE_USER_HAS_NO_PERMISSION, (_payload, { extra }) => {
+  const { navigation, notification } = extra;
+  notification.error(NotificationMessage.PERMISSION_DENIED);
+  navigation.push(AppRoute.SIGN_IN);
```

Comment:
Can't redirection be made also inside a component?

Reply:
agreed. you can also use `notify` action to create a notification. so you don't need this action at all